cmd/generate.go

```diff
@@ -59,6 +61,7 @@ func NewGenerateCommand() *cobra.Command {
 			}
 
 			v := viper.New()
+			viper.Set("verboseOutput", verboseOutput)
```

Comment:
Using the global singleton Viper here since the verbose output setting isn't specific to any backend or even any part of the code (it's used when doing auth, replacements in the template, and when retrieving from backends)